src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
I see, I misunderstood what you meant by pseudo-coefficient. I thought you meant something like the full-rank dummy coefficient, not the analogous coefficient given the coding scheme. That might make sense for something like dummy or effects coding, but what would it be for helmert, or forward/reverse difference coding (which we don't currently implement but might as well)?

In general, I'm a little wary of reporting anything other than the actual fitted values _by default_. I think, however, that we should definitely provide a way of "re-coding" fitted models on the fly (since as long as the contrasts are well-formed than any system is linearly equivalent to any other). But it seems to me that that's something users should have to ask for.